libraries/AP_Compass/AP_Compass_Calibration.cpp

```diff
@@ -230,32 +230,37 @@ bool Compass::_accept_calibration_mask(uint8_t mask)
 
 bool Compass::send_mag_cal_progress(const GCS_MAVLINK& link)
 {
-    uint8_t cal_mask = _get_cal_mask();
```

Comment:
Why the inline of this?

Reply:
Because we only use it once, and grabbing it early is actually a deoptimization if not calibrating.